modules/exploits/windows/fileformat/device_viewer.rb

```diff
@@ -0,0 +1,204 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::FILEFORMAT
+  include Msf::Exploit::Remote::Seh
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'    => 'Shenzhen Sricctv Technology DeviceViewer User Field Stack Buffer Overlow',
+      'Description'  => %q{
+        This exploits a SEH stack buffer overflow in DeviceViewer v.3.10.12.0 present in the
```

Comment:
`Shenzhen Sricctv Technology DeviceViewer`. Might also want to change this in your Scenarios section of your documentation as well.

Reply:
now fixed :)

Reply:
Confirmed fixed, marking as resolved :)